code/Kevin/Python/lab02a.py

```diff
@@ -56,10 +68,19 @@
         print('Your input was not a number. Please try again.')
         time.sleep(2)
 
+while True:
+    user_input_3 = input(prompt_3)
+    if user_input_3 in distance_dictionary:
+        break
+    else:
+        print("You did not type an appropriate response. Please try again.")
+        time.sleep(2)
+
 #Multiplies the user input by the value stored in the correct key.
 math_time = user_input_2 * distance_dictionary[user_input_1]
```

Comment:
I believe there may be a small error somewhere around here in your formula. Take a look at it again. Make sure you run your program with different inputs to test.